langstream-api-gateway/pom.xml

```diff
@@ -112,6 +112,11 @@
 			<artifactId>kafka</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>pulsar</artifactId>
```

Comment:
please note that this dependency imports some version of the Pulsar client, is it polluting the classpath ?

Reply:
It seems to be OK. Testcontainers uses the same shaded jar that the runtime uses.